testproject/jijtest/build.gradle

```diff
@@ -40,7 +40,9 @@ test {
 }
 
 neoForge {
-    version = project.neoforge_version
+    enableModding {
+        neoForgeVersion = project.neoforge_version
+    }
```

Comment:
I think this looks a bit weird since this is already the `neoforge` block. I think we need to discuss how the interfaces for the normal and legacy extensions should look like.